integrations/gradle-bloop/src/main/scala/bloop/integrations/gradle/BloopParameters.scala

```diff
@@ -6,64 +6,47 @@ import org.gradle.api.Project
 import syntax._
 
 /**
-  * Project extension for bloop-specific configurations
-  *
-  * For each property we have a private variable and a set of interface functions to support convenient access
-  * from Scala, Java and the Groovy DSL.
-  *
-  * From the build definitions it can be used like the following:
-  *
-  * {{{
-  * bloop {
-  *   targetDir "$projectDir/.bloop"
-  *   mainSourceSet "main"
-  *   compilerName "scala-compiler"
-  * }
-  * }}}
-  */
-class BloopParameters(project: Project) {
+ * Project extension to configure Bloop.
+ *
+ * For each property we have a private variable and a set of interface functions to support
+ * convenient access from Scala, Java and the Groovy DSL.
+ *
+ * From the build definitions it can be used like the following:
+ *
+ * {{{
+ * bloop {
+ *   targetDir "$projectDir/.bloop"
+ *   mainSourceSet "main"
+ *   compilerName "scala-compiler"
+ * }
+ * }}}
+ */
+final class BloopParameters(project: Project) {
```

Comment:
Seems like it cannot be final, because of some unexpected Gradle magic :)

```
Caused by: org.gradle.api.GradleException: Could not generate a proxy class for class bloop.integrations.gradle.BloopParameters.
        at org.gradle.api.internal.AbstractClassGenerator.generateUnderLock(AbstractClassGenerator.java:201)
        at org.gradle.api.internal.AbstractClassGenerator.generate(AbstractClassGenerator.java:64)
        at org.gradle.api.internal.ClassGeneratorBackedInstantiator.newInstance(ClassGeneratorBackedInstantiator.java:36)
        at org.gradle.api.internal.plugins.DefaultConvention.create(DefaultConvention.java:103)
        at bloop.integrations.gradle.syntax$ProjectExtension.createExtension(syntax.scala:42)
        at bloop.integrations.gradle.BloopPlugin.apply(BloopPlugin.scala:20)
        at bloop.integrations.gradle.BloopPlugin.apply(BloopPlugin.scala:17)
        at org.gradle.api.internal.plugins.ImperativeOnlyPluginApplicator.applyImperative(ImperativeOnlyPluginApplicator.java:35)
        at org.gradle.api.internal.plugins.RuleBasedPluginApplicator.applyImperative(RuleBasedPluginApplicator.java:43)
        at org.gradle.api.internal.plugins.DefaultPluginManager.doApply(DefaultPluginManager.java:137)
        ... 124 more
Caused by: java.lang.VerifyError: Cannot inherit from final class
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:642)
        at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:75)
        at org.gradle.api.internal.AsmBackedClassGenerator$ClassBuilderImpl.generate(AsmBackedClassGenerator.java:908)
        at org.gradle.api.internal.AbstractClassGenerator.generateUnderLock(AbstractClassGenerator.java:199)```